src/CodeGeneration.Roslyn.Tool/Program.cs

```diff
@@ -1,11 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using CodeGeneration.Roslyn.Tool.CommandLine;
-using Microsoft.CodeAnalysis;
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This isn't a Microsoft copyrighted work.

Reply:
Fixed